src/mono/wasm/Wasm.Build.Tests/BlazorWasmProjectProvider.cs

```diff
@@ -0,0 +1,61 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.IO;
+using System.Linq;
+using Xunit;
+using Xunit.Abstractions;
+using System.Runtime.Serialization.Json;
+using Microsoft.NET.Sdk.WebAssembly;
+using System.Text.Json.Nodes;
+
+#nullable enable
+
+namespace Wasm.Build.Tests;
+
+public class BlazorWasmProjectProvider(string projectDir, ITestOutputHelper _testOutput)
+                : ProjectProviderBase(projectDir, _testOutput)
+{
+    public void AssertBlazorBootJson(string config, bool isPublish, bool isNet7AndBelow, string targetFramework, string? binFrameworkDir)
+    {
```

Comment:
Blazor boot json schema is now shared with WasmAppBuilder. We should do the same check for every app bundle

Reply:
And the same thing is needed for checking icu files.